### DISCUSSION TITLE
Is there has the finished phantom which was approved by committee?

### DISCUSSION THREAD

**Discussion by kstcmcjw** [nrc-cnrc/EGSnrc#873] [Questions] [answered]
Hello everyone, i want to know is there has the finished phantom which was approved by committee?
Those days i learned BEAMnrc, but a real problem i have just met, and i have some problems need you help.
The treatment of liver cancer has several methods, surgery, radiotherapy, interventional therapy and so on. 
Element Y-90 (yttrium), the interventional therapy of Y-90 glass microspheres is used for non-surgery who is terminally patiant, i want to 
 simulation a patiant who has implant the Y-90 glass microspheres, in order to get the operator's equivalent dose and the some specifict location's dose rate.
There are some questions i must face:
1. How to simulation a patiant? With an ICRU ball? but i find the density files of muscle and skeletal in the $HEN HOUSE\pegs4\density_corrections\compounds, how to simulation a patiant, whether there has a complished phantom?

 
![image](https://user-images.githubusercontent.com/96934217/168214740-90a4e29b-9f6a-48fc-8009-4f267af961c7.png)

2. In the EGS getting start, some example simulation a particle's energy but not the element's activty, the activtiy of  Y-90 glass microspheres'  interventional therapy is 7×10^9，how to simulation a 'particular activity' radiactive source?

Thank you for you answer, wish you have a nice day !

**Comment by rtownson** [comment, marked as answer]
Most users use CT DICOM data to model patients, converted to the EGSnrc egsphant format using ctcreate. There is a standard human phantom, but it is not compatible with EGSnrc out of the box. You may be able to find others in the literature who have done this, and obtain their egsphant files:

https://www.icrp.org/publication.asp?id=icrp%20publication%20110

Another human phantom model is supported in a special format here, but again it must be obtained from the original author (referenced in the comments):
https://github.com/nrc-cnrc/EGSnrc/blob/b735e209236878963c2133f08a77c10c79f12308/HEN_HOUSE/egs%2B%2B/geometry/egs_vhp_geometry/example.geom

Radionuclides can be modelled in egs++ using the radionuclide source, or you could just use a spectrum source.

https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadionuclideSource.html

The activity does not impact the simulation dose normalization, because results are reported 'per decay'. You can use the activity to normalize the results afterward. You may want to take a look at egs_brachy for some extra functionality:

https://physics.carleton.ca/clrp/egs_brachy

**Comment by kstcmcjw** [comment]
Oh, yes! It's very kind and friend of you to reply my question! First of all, thank you very much! I am sorry for reply so late!

 I read your reply carefully, but i didn't simulation the phantom of "EGSnrc/HEN_HOUSE/egs++/geometry/egs_vhp_geometry/example.geom" successfully using the EGS_view software, er, i will go on study the geometry.
I really feel very strong of EGS, because its has subbranch, such as BEAM and brachy, EGS is a treasure-house, I must study carefully from now on.

**Reply by rtownson** [reply]
EGSnrc does not include all the data for this example geometry, you would need to contact R. Kramer if you would like to obtain his phantom. The phantom you might be interested in is the FAX06 or MAX06:

https://pubmed.ncbi.nlm.nih.gov/17148818/

You would need to obtain for example fax06_med1.bin and the micro-matrix data. Note that this is a very detailed phantom and may be overkill for most applications. You might find it easier to obtain a CT scan of an artificial human phantom and convert it to the 'egsphant' format using ctcreate.
